src/main/java/minicraft/entity/mob/RemotePlayer.java

```diff
@@ -18,7 +17,7 @@
 import minicraft.level.Level;
 
 /** This is used for players in multiplayer mode. */
-public class RemotePlayer extends Player implements ClientTickable {
+public class RemotePlayer extends Player {
```

Comment:
Are you sure this won't break something?

Reply:
Player already implements ClientTickable, so this is unneeded verbosity imo